### PR TITLE
Add support for env vars in phpunit.xml.*

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: tests/Configurator/EnvConfiguratorTest.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+         colors="true"
+         bootstrap="vendor/autoload.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+        <server name="KERNEL_CLASS" value="App\Kernel" />
+
+<!-- ###+ FooBundle ### -->
+<server name="APP_ENV" value="test" />
+<server name="APP_DEBUG" value="0" />
```
Comment (yceruto) [inline tests/Configurator/EnvConfiguratorTest.php]: Could be aligned with default indent 8, regarding to default file structure.

Comment (fabpot) [reply tests/Configurator/EnvConfiguratorTest.php]: done

File: src/Configurator/EnvConfigurator.php [hunk 2]
```diff
@@ -47,15 +59,44 @@ public function configure(Recipe $recipe, $vars): void
...
+            $data = '';
+            foreach ($vars as $key => $value) {
+                if ('%generate(secret)%' === $value) {
+                    $value = bin2hex(random_bytes(16));
+                }
+                if ('#' === $key[0]) {
+                    $data .= '        <!-- '.$value." -->\n";
+                } else {
+                    $value = $this->options->expandTargetDir($value);
+                    $data .= "        <server name=\"$key\" value=\"$value\" />\n";
```
Comment (yceruto) [inline src/Configurator/EnvConfigurator.php]: According to https://github.com/symfony/recipes/pull/170 shouldn't be `<env ... />`?

Comment (fabpot) [reply src/Configurator/EnvConfigurator.php]: indeed, fixed. Thanks.

**Pull request merged**